hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedTableMetadata.java

```diff
@@ -411,6 +411,61 @@ public void testRepeatedCleanActionsWithMetadataTableEnabled(final HoodieTableTy
     assertEquals(tableVersion, finalTableVersion.versionCode());
   }
 
+  @ParameterizedTest
+  @CsvSource({"COPY_ON_WRITE,6", "COPY_ON_WRITE,8", "MERGE_ON_READ,6", "MERGE_ON_READ,8"})
+  void testDeletePartitionKeyOnCleanPartition(final HoodieTableType tableType, int tableVersion) throws Exception {
```

Comment:
we dont' need to make this run for 6 combinations. lets not try to add more parametrized test if there is no need. 
this is just testing regular updates to FILES partition in MDT. 
just 1 version would suffice.